ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/router-bgp-base.cfg

```diff
@@ -97,20 +97,27 @@ router bgp 65101
       neighbor foo prefix-list PL-BAR-v4-IN in
       neighbor foo prefix-list PL-BAR-v4-OUT out
       neighbor foo default-originate route-map RM-FOO-MATCH always
+      neighbor 10.2.3.8 rcf in Address-Family_IPV4_In()
```

Comment:
Not relevant to molecule but i think there is some limitation on function name. hyphen(-) is not supported for function name. Please do check other chars

Reply:
Ohh.. we can change it, I just kept a dummy name for now. Once we are able to create RCF with AVD then may be we can link the function names.
For now I will replace hyphen(-) with underscore(_).

Reply:
Done!